### PR TITLE
AddonManager: tooltips and minor code formatting

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: src/Mod/AddonManager/addonmanager_workers.py [hunk 1]
```diff
@@ -39,16 +39,24 @@
...
+MACROS_BLACKLIST = ["BOLTS",
+                    "WorkFeatures",
+                    "how to install",
+                    "PartsLibrary",
+                    "FCGear"]
+
+# These addons will print an additional message informing the user
+OBSOLETE =         ["assembly2",
+                    "drawing_dimensioning",
+                    "cura_engine"]
```
Comment (luzpaz) [inline src/Mod/AddonManager/addonmanager_workers.py]: @yorikvanhavre is this OK with you?

File: src/Mod/AddonManager/AddonManagerOptions.ui [hunk 2]
```diff
@@ -17,7 +17,9 @@
    <item>
     <widget class="QCheckBox" name="checkUpdates">
      <property name="toolTip">
-      <string>If this option is checked, when launching the Addon Manager, installed addons will be checked for available updates (requires the python-git package installed on your system)</string>
+      <string>If this option is selected, when launching the Addon Manager,
+installed addons will be checked for available updates
+(this requires the python-git package installed on your system)</string>
      </property>
      <property name="text">
       <string>Automatically check for updates at start (requires python-git)</string>
```
Comment (vocx-fc) [inline src/Mod/AddonManager/AddonManagerOptions.ui]: I would use the name GitPython, which is the actual name of the project. The name `python-git` refers to a package, which may be different or non-existent in other Linux distributions.

Comment (luzpaz) [reply src/Mod/AddonManager/AddonManagerOptions.ui]: To be fair, per [repology](https://repology.org/project/python:gitpython/versions) python-git is fairly popular synonym for GitPython and so this is somewhat trivial, but nevertheless aeaa3a1

File: src/Mod/AddonManager/addonmanager_workers.py (hunk 1 above)
Comment (yorikvanhavre) [reply src/Mod/AddonManager/addonmanager_workers.py]: sure!

**Pull request merged**